lms/djangoapps/student_account/views.py

```diff
@@ -147,6 +147,11 @@ def login_and_registration_form(request, initial_mode="login"):
         ),
     }
 
+    enterprise_customer_uuid = request.COOKIES.get(settings.ENTERPRISE_CUSTOMER_COOKIE_NAME)
+    if enterprise_customer_uuid:
+        request.GET = request.GET.copy()
+        request.GET['enterprise_customer'] = enterprise_customer_uuid
```

Comment:
Move the cookie check to the [enterprise_support](https://github.com/edx/edx-platform/blob/master/openedx/features/enterprise_support/api.py#L235) module.